includes/class-wc-payments.php

```diff
@@ -591,14 +591,16 @@ public static function init() {
 		require_once __DIR__ . '/migrations/class-allowed-payment-request-button-types-update.php';
 		require_once __DIR__ . '/migrations/class-allowed-payment-request-button-sizes-update.php';
 		require_once __DIR__ . '/migrations/class-update-service-data-from-server.php';
-		require_once __DIR__ . '/migrations/class-track-upe-status.php';
 		require_once __DIR__ . '/migrations/class-delete-active-woopay-webhook.php';
 		add_action( 'woocommerce_woocommerce_payments_updated', [ new Allowed_Payment_Request_Button_Types_Update( self::get_gateway() ), 'maybe_migrate' ] );
 		add_action( 'woocommerce_woocommerce_payments_updated', [ new \WCPay\Migrations\Allowed_Payment_Request_Button_Sizes_Update( self::get_gateway() ), 'maybe_migrate' ] );
 		add_action( 'woocommerce_woocommerce_payments_updated', [ new \WCPay\Migrations\Update_Service_Data_From_Server( self::get_account_service() ), 'maybe_migrate' ] );
-		add_action( 'woocommerce_woocommerce_payments_updated', [ '\WCPay\Migrations\Track_Upe_Status', 'maybe_track' ] );
 		add_action( 'woocommerce_woocommerce_payments_updated', [ '\WCPay\Migrations\Delete_Active_WooPay_Webhook', 'maybe_delete' ] );
 
+		// Delete UPE admin notes as UPE is the default now. This is part of the migration and should be removed in the future.
+		require_once WCPAY_ABSPATH . 'includes/notes/class-wc-payments-notes-additional-payment-methods.php';
+		WC_Payments_Notes_Additional_Payment_Methods::possibly_delete_note();
```

Comment:
I know we're also currently set up to use "migrations" that run on plugin upgrade. Is there a limitation from running the `possibly_delete_note` on upgrade, rather than on init (so that the query doesn't have to run on each page request)? (I didn't see this in the explanation you provided, and wanted to get additional clarification)

Reply:
Thank you for pointing this out, this is definitely the cleaner option to treat this deletion as part of the migration. I've added one in https://github.com/Automattic/woocommerce-payments/commit/6722c2d89670ad51422b409e3e65a2ce77d87990